### PR TITLE
rpc needs cppo_ocamlbuild

### PR DESCRIPTION
**Pull request by avsm**
from check.ocamlabs.io

### CODE REVIEW

**Comment by mseri** [comment]
I think it only needs it for `cppo > 1.5.0`. I need to check how one adds the "or" dependency

**Comment by kit-ty-kate** [comment]
The tests fail:
```
#=== ERROR while compiling rpc.1.9.52 =========================================#
# context              2.0.4 | linux/x86_64 | ocaml-base-compiler.4.04.2 | pinned(https://github.com/mirage/ocaml-rpc/archive/v1.9.52.tar.gz)
# path                 ~/.opam/4.04/.opam-switch/build/rpc.1.9.52
# command              /usr/bin/make test
# exit-code            2
# env-file             ~/.opam/log/rpc-15-601c21.env
# output-file          ~/.opam/log/rpc-15-601c21.out
### output ###
# make -C ppx_test
# make[1]: Entering directory '/home/opam/.opam/4.04/.opam-switch/build/rpc.1.9.52/ppx_test'
# ocamlfind ocamlopt -package ppx_deriving -ppxopt ppx_deriving,../_build/ppx/ppx_deriving_rpc.cma -linkpkg -package lwt,lwt.unix,oUnit,result,rresult -I ../_build/lib rpclib_core.cmxa rpclib.cmxa json.cmxa -o test -dsource test.ml 2> test.gen.ml
# Makefile:16: recipe for target 'test' failed
# make[1]: *** [test] Error 2
# make[1]: Leaving directory '/home/opam/.opam/4.04/.opam-switch/build/rpc.1.9.52/ppx_test'
# Makefile:44: recipe for target 'test' failed
# make: *** [test] Error 2
```
Sounds like missing dependencies (probably ounit at least)

**Comment by mseri** [comment]
@kit-ty-kate Don't look at the Makefile that used to run those tests 🙄 ... I suggest we disable them.

Ping @gaborigloi: even if the new interface is still transient, I think we should publish the latest version. It is much more stable and polished.

File: packages/rpc/rpc.1.9.51/opam [hunk 1]
```diff
@@ -23,7 +21,7 @@ remove: [
 depends: [
   "ocaml" {< "4.06.0"}
   "oasis" {build}
-  "cppo" {build}
+  "cppo" {build & < "1.5.0"} | "cppo_ocamlbuild" {build & "cppo" >= "1.5.0"}
```
Comment (kit-ty-kate) [inline packages/rpc/rpc.1.9.51/opam]: @mseri this doesn't work (`"cppo" >= "1.5.0"` is not a valid attribute). Why not just ask for `cppo_ocamlbuild`? There is no packages in opam-repository right now that requires `cppo < 1.6.0`. Except being (too?) precise, it doesn't add any benefits imo.

**Comment by gaborigloi** [comment]
Sounds good, we should somehow mark the new version as pre-release. I haven't looked at these things in a while, so maybe it's best if someone more active releases it ping @edwintorok  @jonludlam @rwdrich @psafont

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**